### PR TITLE
Add CDF for Laplace distribution

### PR DESCRIPTION
**Pull request by tonyduan**
Adding a CDF for the Laplace distribution.

More or less follows the implementation for the Normal distribution:
https://github.com/google/jax/blob/master/jax/scipy/stats/norm.py

Happy to take any suggestions, since this is my first time contributing -- thanks!

### CODE REVIEW

**Comment by tonyduan** [comment]
I signed it!

File: jax/scipy/stats/laplace.py [hunk 1]
```diff
@@ -33,3 +33,16 @@ def logpdf(x, loc=0, scale=1):
...
   return lax.exp(logpdf(x, loc, scale))
+
+@_wraps(osp_stats.laplace.cdf)
+def cdf(x, loc=0, scale=1):
+  x, loc, scale = _promote_args_like(osp_stats.laplace.cdf, x, loc, scale)
+  half = _constant_like(x, 0.5)
+  one = _constant_like(x, 1)
+  zero = _constant_like(x, 0)
+  diff = lax.div(lax.sub(x, loc), scale)
+  case1 = lax.mul(lax.mul(half, lax.exp(diff)),
```
Comment (hawkinsp) [inline jax/scipy/stats/laplace.py]: Is there a reason not to write this using `lax.select`, like this:
```
...
return lax.select(lax.ge(diff, zero),
                  lax.mul(half, lax.exp(diff)),
                  lax.sub(one, lax.mul(half, lax.exp(lax.neg(diff)))
```
?

**Review by hawkinsp — COMMENTED** [review]
Thanks for the PR! Looks pretty good.

Would you mind also adding a test to `scipy_stats_test.py` as well? Tests are the easiest way to make sure this *stays* working now you have added it...

**Comment by tonyduan** [comment]
Thanks for the suggestions! I've added tests to `scipy_stats_tests.py` and updated the implementation to use `lax.select`. PTAL.

**Comment by hawkinsp** [comment]
Thanks for the PR!

**Pull request merged**